3467/3/db/migrate/20190724140813_create_recalls.rb

```diff
@@ -0,0 +1,12 @@
+#:reek:FeatureEnvy
+class CreateRecalls < ActiveRecord::Migration
+  def change
+    create_table :recalls do |ttt|
+      ttt.belongs_to :shop, index: true
+      ttt.text :comment
+      ttt.integer :rating
+
+      ttt.timestamps
```

Comment:
DEPRECATION WARNING: `#timestamps` was called without specifying an option for `null`. In Rails 5, this behavior will change to `null: false`. You should manually specify `null: true` to prevent the behavior of your existing migrations from changing. (called from block in change at summer-2019/3467/3/db/migrate/20190724140813_create_recalls.rb:9)

Reply:
На указание вручную null: true ругается rubocop. При этом выставленные в миграциях дизэйблы не отключают rubocop ни для строки, ни для нескольких строк. Как я понял, тут нужно менять конфиг rubocopa: configure using TargetRubyVersion parameter, under AllCops

Reply:
Так это ж сообщение из рельсов. И что значит ругается рубокоп? Ты не написал какое именно сообщение выводится. Или ты их не читаешь?

Reply:
Rubocop выдаёт нарушение:
db/migrate/20190724133854_create_shops.rb:10:23: E: Lint/Syntax: unexpected token tLABEL
(Using Ruby 2.3 parser; configure using TargetRubyVersion parameter, under AllCops)
      ttt.timestamps, null: true
Советы с разных сайтов не решают вопрос.

Reply:
Да и в этом PR эта ошибка есть: https://github.com/rubizza-camp/summer-2019/pull/492#discussion_r308977206